### PR TITLE
Catalog of my things: Item class

### PR DESCRIPTION
**Pull request by zhorabay**
- [ ] Description will be updated later

### CODE REVIEW

**Comment by zhorabay** [comment]
Dear Code reviewer,
We have linter errors because we have created app.rb and main.rb files as it is asked in the project requirements however we didn't implement anything there because the project hasn't been finished yet. So, please accept this PR for now and we will implement methods in the future to avoid linter errors.
Thank you in advance!

**Comment by Chimoneg27** [comment]
Please perform the review

File: classes/item.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  def label=(label)
+    @label = label
+    label.items << self unless label.items.include?(self)
+  end
+
+  def can_be_archived()
+    difference = Time.now.year - @publish_date.year.move_to_i
+    difference >= 10
+  end
+end
```
Comment (jubaer919) [inline classes/item.rb:32]: - You should create a move_to_archive() method in this class

**Review by jubaer919 — CHANGES_REQUESTED** [review]
♻️ Require Changes

- please fix you linter error

File: classes/item.rb (hunk 1 above)
Comment (Chimoneg27) [reply classes/item.rb:32]: Alright I'll do so

**Review by jubaer919 — APPROVED** [review]
# Approved 👏 

- There is nothing to say other then marge it.

**Pull request merged**